<launch>
  <arg name="bag_path"/>
  <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 1.58 1.58   initial_pose map 100" />
  <node name="mono_orbslam2" pkg="orb_slam2" type="mono" output="screen">
  <param name="voc_path" value="/home/jpiat/development/SOFTWARE/ORB_SLAM2/Vocabulary/ORBvoc.txt"/>
  <param name="config_path" value="/home/jpiat/development/SOFTWARE/ORB_SLAM2/Examples/Monocular/KITTI04-12.yaml"/>
  </node>
  <node name="rviz_orbslam2" pkg="rviz" type="rviz" args="-d $(find orb_slam2)/launch/orbslam2.rviz"/>
  <node pkg="rosbag" type="play" name="player" output="screen" args="--clock $(arg bag_path)"/>
</launch>
